pkg/database/sync.go

```diff
@@ -20,8 +20,10 @@ func (dao *DAO) SyncData(event model.SyncEvent, clusterName string) {
 
 	// ADD
 	for _, resource := range event.AddResources {
-		json, _ := json.Marshal(resource.Properties)
-		batch.Queue("INSERT into search.resources values($1,$2,$3)", resource.UID, clusterName, string(json))
+		data, _ := json.Marshal(resource.Properties)
+		p := resource.Properties
+		json.Unmarshal([]byte(data), &p)
+		batch.Queue("INSERT into search.resources values($1,$2,$3,$4,$5,$6)", resource.UID, clusterName, p, p.Kind, p.NameSpace, p.Name)
```

Comment:
Should we remove `kind, namespace, name` from the json data `p` ?

**Pro:** Eliminate data duplication.
**Cons:** Queries will be more complex when including these properties.

Reply:
An alternative could be to use jsonb indexing on these keys to make searching for these more efficient.
https://www.postgresql.org/docs/current/datatype-json.html#JSON-INDEXING

Reply:
Looking at the docs, we would have to change our queries to use the `@>` operator to make use of the jsonb indexing.

Reply:
for example: `select data from search.resources where data @> '{"kind" : "Pod"}';`

Reply:
or `select data->'namespace' from search.resources where data @> '{"kind" : "Pod"}';`